### DISCUSSION TITLE
Issue with file permissions

### DISCUSSION THREAD

**Discussion by brobb1972** [iterate-ch/cyberduck#15082] [Q&A]
Hi, 

I have a customer who uses Cyperduck to access our SFTP server.  (CoreFTP).  When he logs in, all of his folders have the red "not" sign attached to them.  See attached.  I tried it and get the same thing, however, it does not prevent me from uploading/downloading/moving/renaming etc.  The permissions allow him to do everything but execute.  I tried accessing the same account in FileZilla and it does not show these icons.  

Is this a bug, or is there something I am missing. 

![SS-2023-08-30 at 16-49-39@2x](https://github.com/iterate-ch/cyberduck/assets/48924841/a48dd378-311c-44be-924a-3ce2bb266772)

**Comment by dkocher** [comment]
Missing execute permissions on a folder indicate the folder cannot be opened and is therefore marked with the no entry badge.
